code/datums/mood_events/generic_negative_events.dm

```diff
@@ -295,3 +295,13 @@
 	description = "<span class='warning'>I really don't like when people touch me.</span>\n"
 	mood_change = -5
 	timeout = 4 MINUTES
+
+/datum/mood_event/noogie
+	description = "<span class='warning'>Ow! This is like space high school all over again...</span>\n"
+	mood_change = -2
+	timeout = 1 MINUTES
+
+/datum/mood_event/noogie_harsh
+	description = "<span class='warning'>OW!! That was even worse than a regular noogie!</span>\n"
+	mood_change = -4
```

Comment:
i'm going to noogie all the moths i know no matter what. you cant and will not stop me

Reply:
fuuuuck. shiiit. FUCK. daaaamnn......

Reply:
> i'm going to noogie all the moths i know no matter what. you cant and will not stop me

how are you going to noogie yourself (without becoming a slimeperson)??

Reply:
i'll simply spawn a 2nd of me with admin powers